network-api/networkapi/wagtailpages/templates/wagtailpages/research_authors_index_page.html

```diff
@@ -1,15 +1,62 @@
 {% extends "./research_base.html" %}
-{% load profile_tags %}
-(
+{% load profile_tags wagtailimages_tags i18n static %}
+
+{% block hero_guts %}
+  <div id="hero">
+    <div class="banner tw-bg-black">
+      {% with banner=page.banner_image %}
+        {% if banner %}
+          <picture>
+            <source media="(min-width: 1200px)" srcset="{% image_url banner "fill-4032x1152" %}">
+            <source media="(min-width: 992px)" srcset="{% image_url banner "fill-2400x686" %}">
+            <source media="(min-width: 768px)" srcset="{% image_url banner "fill-1984x567" %}">
+            <source media="(min-width: 576px)" srcset="{% image_url banner "fill-1536x439" %}">
+            <img src="{% image_url banner "fill-1536x439" %}" alt="">
+          </picture>
+        {% else %}
+          <img src="{% static "_images/banner-black-white-marble.jpg" %}" alt="">
+        {% endif %}
+      {% endwith %}
+    </div>
+  </div>
+{% endblock %}
+
 {% block subcontent %}
-  <h1>
-    {{ page.title }}
-  </h1>
-  {% if author_profiles %}
-    <ul>
-      {% for author_profile in author_profiles  %}
-        <li><a href="{% routableprofileurl page=page profile=author_profile url_name="author_detail"  %}">{{ author_profile.name }} ({{ author_profile.locale.language_code }})</a></li>
-      {% endfor %}
-    </ul>
-  {% endif %}
+  {# The relative position is necessary to allow this container to overflow the banner image #}
+  <div class="tw-relative tw-w-auto tw-max-w-screen-large xlarge:tw-max-w-screen-xlarge tw-mt-[7rem] tw-mr-4 small:tw-mr-5 medium:tw-mr-7 large:tw-mr-auto tw-ml-0 large:tw-ml-auto tw-pt-4 medium:tw-pt-5 large:tw-pt-7 tw-pr-0 large:tw-pr-5 small:tw-pb-[3rem] tw-pl-4 small:tw-pl-5 medium:tw-pl-7 large:tw-pl-5 tw-bg-white">
+    <div>
+      <div class="tw-font-zilla">
+        {# Replace this with real breadcrumbs #}
+        Research /
+      </div>
+      <h1>{% trans "Browse Authors" %}</h1>
+    </div>
+
+      {% if author_profiles %}
+        <ul class="tw-grid medium:tw-grid-cols-2 medium:tw-gap-6 tw-gap-5 tw-gap-y-7 tw-list-none tw-p-0">
+          {% for author_profile in author_profiles %}
+            <li class="tw-m-0">
+              <a
+                href="{% routableprofileurl page=page profile=author_profile url_name="author_detail" %}"
+                class="tw-flex large:tw-gap-6 medium:tw-gap-5 tw-gap-4 tw-items-center hover:tw-no-underline"
+                >
+                <div class="large:tw-w-[97px] medium:tw-w-[68px] tw-w-[77px]">
+                  {% image author_profile.image fill-96x96 as profile_img %}
+                  <img src="{{ profile_img.url }}" alt="{{ author_profile.name }}" class="tw-w-full tw-h-auto tw-rounded-full">
+                </div>
```

Comment:
**Issue** Sorry I missed this one before. Profiles are not required to have an image. We should only show the image for those that have. 

Maybe we can also update the profile factory to set an image on the profiles? I think most will, but we should handle the case where they don't.